### ISSUE TITLE
AttributeError: can't set attribute with Django 1.9 and Py3

### ISSUE THREAD

**Issue by jdufresne** [alex/django-taggit#351]
I have a model using taggit like: `tags = taggit.managers.TaggableManager()`

During `django.setup()` the following error occurs in taggit:

```
$ venv/bin/python manage.py collectstatic --noinput
Traceback (most recent call last):
  File "manage.py", line 9, in <module>
    execute_from_command_line(sys.argv)
  File ".../venv/lib64/python3.4/site-packages/django/core/management/__init__.py", line 350, in execute_from_command_line
    utility.execute()
  File ".../venv/lib64/python3.4/site-packages/django/core/management/__init__.py", line 324, in execute
    django.setup()
  File ".../venv/lib64/python3.4/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File ".../venv/lib64/python3.4/site-packages/django/apps/registry.py", line 108, in populate
    app_config.import_models(all_models)
  File ".../venv/lib64/python3.4/site-packages/django/apps/config.py", line 202, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/lib64/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2254, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2226, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1471, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  ...
  File ".../app/models.py"
    tags = taggit.managers.TaggableManager()
  File ".../venv/lib64/python3.4/site-packages/taggit/managers.py", line 285, in __init__
    rel = TaggableRel(self, related_name, self.through, to=to)
  File ".../venv/lib64/python3.4/site-packages/taggit/managers.py", line 55, in __init__
    self.to = to
AttributeError: can't set attribute
```

**Comment by jdufresne** [comment]
Looking at the Django 1.9 code, I see `ForeignObjectRel.to()` has the following comment:

```
    # Some of the following cached_properties can't be initialized in
    # __init__ as the field doesn't have its model yet. Calling these methods
    # before field.contribute_to_class() has been called will result in
    # AttributeError
    @property
    def to(self):
        warnings.warn(
            "Usage of ForeignObjectRel.to attribute has been deprecated. "
            "Use the model attribute instead.",
            RemovedInDjango20Warning, 2)
        return self.model
```

**Comment by frewsxcv** [comment]
There are a few 1.9 pull requests open. If you point me to one that fixes this problem, I'd be happy to merge it

**Comment by jdufresne** [comment]
PRs  #329 and #343 look to be handling the issue (untested on my end). The correct area of code has be modified. #329 looks more complete as far as 1.9 is concerned. #343 targets only this specific issue. No strong opinion on which one is better quality wise.

I can try testing against these PRs and report back.

**Comment by frewsxcv** [comment]
> I can try testing against these PRs and report back.

Awesome, thanks :)

**Comment by jdufresne** [comment]
After some testing and sanity checks, both PRs fix the problem for me and don't appear to have any obvious bugs. From my perspective either one works. Whatever looks best to for the taggit project sounds good to me.

**Comment by frewsxcv** [comment]
Just released 0.17.2 that has Django 1.9 fixes

https://pypi.python.org/pypi/django-taggit/0.17.2
